modulefiles/ufs_aerosols_hera.intel

```diff
@@ -0,0 +1,9 @@
+#%Module
+
+module use -a /scratch1/NCEPDEV/nems/Raffaele.Montuoro/dev/nasa/ufs/lib/modulefiles
```

Comment:
No personal directories. These three modules must be installed in the same location as other 'common' modules and this should  be moved to `modulefiles/ufs_common`

Reply:
I've submitted PR [#252](https://github.com/NOAA-EMC/hpc-stack/pull/252) to add these dependencies to hpc-stack.

Reply:
Can we not add/maintain this model file, instead just add those modules to hera.intel.

Reply:
This modulefile has been removed (see commit e9b701446cb3ccfd43499ef6038902ae8f5df3c6)

Reply:
The file was deleted in commit 8de4676cffdbbd7d6462d2b814688ca5076ff081